### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by adith-a-danthi**
## Overview

- Add [tidle-ui](https://tidle-ui.netlify.app/)
- Add react router
- Added Navbar - Add Homepage
- Updated categories db
- Add Homepage

### CODE REVIEW

File: src/components/CategoriesSection.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      console.log(error);
+    }
+  }, []);
+
+  return (
+    <div className="grid grid-cols-3 fluid-grid gap-1">
+      {categories.map(({ _id, categoryImage, categoryName }) => (
+        <div className="card" key={_id}>
+          <a href="#">
+            <img src={categoryImage} alt="" className="card-img fill-height" />
```
Comment (samyakshah3008) [inline src/components/CategoriesSection.jsx]: You can fill up alt tag with your image name for good code quality

File: src/styles/_reset.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* Remove default margin */
+* {
+  margin: 0;
+}
+/* Allow percentage-based heights in the application */
```
Comment (samyakshah3008) [inline src/styles/_reset.css:12]: Loved how you have included comments, this gives better review experience

File: src/views/Home/Home.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.hero-content {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
```
Comment (samyakshah3008) [inline src/views/Home/Home.css]: For this, you can use shorthand property for less number of code lines

File: src/views/Home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+              <Link to="/">
+                <button className="btn btn-secondary font-weight-bold">Shop Now</button>
+              </Link>
+            </div>
+          </div>
+        </section>
+        {/* Categories Section */}
+        <section id="categories" className="py-8">
+          <CategoriesSection />
+        </section>
```
Comment (samyakshah3008) [inline src/views/Home/Home.jsx:25]: Good semantic tags ✅

**Review by samyakshah3008 — APPROVED** [review]
Good folder structure and clean UI 
Please check few single comments, you can refactor later on:) 

Small suggestion on UI as well: You can remove text decoration on navbar's nav-pills (optional) 

Rest is all good, loved how you gave description for preview ✅

**Review by ArjunGTX — APPROVED** [review]
Code looks clean and comments made the review easy

**Pull request merged**